svf-llvm/include/SVF-LLVM/LLVMModule.h

```diff
@@ -187,7 +187,8 @@ class LLVMModuleSet
     const Value* getLLVMValue(const SVFValue* value) const
     {
         SVFValue2LLVMValueMap::const_iterator it = SVFValue2LLVMValue.find(value);
-        assert(it!=SVFValue2LLVMValue.end() && "can't find corresponding llvm value!");
```

Comment:
make sure we don't change this method and you can do some pre-check to avoid assertions.

Reply:
Ok